R/evalPars.R

```diff
@@ -69,3 +128,6 @@ getFreeParsRanges <- function(object) {
   })
   par.ranges[free]
 }
+
+#' @import utils
+utils::globalVariables(c("cl", "p"))
```

Comment:
cl is indeed a global variable, and the name is conventional
Ideally p wouldn't be a global variable. I assume it needs to be for foreach to work?

Reply:
It would seem so, because the error I was getting was:

```
evalPars: no visible binding for global variable ‘p’
evalPars : <anonymous>: no visible binding for global variable ‘cl’
```

which was fixed by adding global variables.

[https://github.com/josephguillaume/hydromad/runs/905472912?check_suite_focus=true](https://github.com/josephguillaume/hydromad/runs/905472912?check_suite_focus=true)